setup.cfg

```diff
@@ -25,7 +25,7 @@ install_requires =
     glue-core>=1.2.2
     glue-jupyter>=0.10.1
     echo>=0.5.0
-    ipyvue>=1.4.1
+    ipyvue>=1.6
```

Comment:
Wasn't there another dependency you mentioned on Slack that is needed, or did I misunderstand?

Reply:
Ah, yes, `watchdog` is an optional dependency. It's only needed when a dev wants to enable hot reloading.

Reply:
Should we list `watchdog` under "optional dependencies" or at least document this in the dev doc?